Base/usr/share/man/man2/posix_spawn.md

```diff
@@ -0,0 +1,47 @@
+## Name
+
+`posix_spawn` - launch a new process
+
+## Synopsis
+
+```**c++
+#include <spawn.h>
+
+int posix_spawn(pid_t* pid, const char* file, const posix_spawn_file_actions_t*, const posix_spawnattr_t*, char* const argv[], char* const envp[]);
+int posix_spawnp(pid_t* pid, const char* file, const posix_spawn_file_actions_t*, const posix_spawnattr_t*, char* const argv[], char* const envp[]);
+```
+
+## Description
+
+Spawns a new process reading the binary `file`, passing `argv` as arguments to `main()` and setting `envp` as argument.
+
+Places the pid of the new process in `pid`.
+
+If `file` passed to `posix_spawn` is a relative path, it is resolved relative to the current working directory.
+
+If `file` passed to `posix_spawnp` is a relative path, it is resolved by searching `PATH`.
+
+The `posix_spawn_file_actions_t` and `posix_spawnattr_t` arguments may be `nullptr`. If they aren't, see [`posix_spawn_file_actions`(2)](posix_spawn_file_actions_init.md) and [`posix_spawnattr`(2)](posix_spawnattr_init.md) for what they do.
+
+The last entry in `argv` and `envp` has to be `nullptr`.
+
+To make the subprocess use the parent's environment, declare `extern char* environ;` in your program and pass that.
+
+The new process is started as if the following steps are executed in this order:
+
+1. A new process is started as if `fork()` was called.
+2. If the `posix_spawnattr_t` parameter is non-NULL, its [effects](posix_spawnattr_init.md) are processed.
+3. If the `posix_spawn_file_actions_t` parameter is non-NULL, its [effects](posix_spawn_file_actions_init.md) are processed.
+4. `file` is started as if `execve` or `execvpe` was called.
```

Comment:
We've already "started" the process.

We could say that "the new process begins executing the `file`, as if after an `execve()` call", or something like that.